modules/oe_list_pages_open_vocabularies/tests/src/Kernel/ListPagesSearchApiConfiguratorTest.php

```diff
@@ -161,14 +161,14 @@ public function testVocabularyAssociationChanges(): void {
     $facet_1 = Facet::load($id_1);
     $this->assertEquals('New label', $facet_1->label());
     $this->assertEquals($field_id, $facet_1->getFieldIdentifier());
-    $this->assertEqual(SearchApiConfigurator::MAX_WEIGHT, $facet_1->getWeight());
+    $this->assertEquals(SearchApiConfigurator::MAX_WEIGHT, $facet_1->getWeight());
     $this->assertArrayHasKey('url_processor_handler', $facet_1->getProcessors());
     $this->assertArrayHasKey('display_value_widget_order', $facet_1->getProcessors());
     $this->assertArrayHasKey('translate_entity', $facet_1->getProcessors());
     $facet_2 = Facet::load($id_2);
     $this->assertEquals('New label', $facet_2->label());
     $this->assertEquals($field_id, $facet_2->getFieldIdentifier());
-    $this->assertEqual(SearchApiConfigurator::MAX_WEIGHT, $facet_2->getWeight());
+    $this->assertEquals(SearchApiConfigurator::MAX_WEIGHT, $facet_2->getWeight());
```

Comment:
Looks like also you have to swap the order of arguments (see the signature difference of method).

Reply:
Change updated on next update.